plugins/linear/client/Icon.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+
+type Props = {
+  /** The size of the icon, 24px is default to match standard icons */
+  size?: number;
+  /** The color of the icon, defaults to the current text color */
+  fill?: string;
+};
+
+export default function Icon({ size = 24, fill = "currentColor" }: Props) {
```

Comment:
The logo is a bit large in the space, I can fix as we get closer to merging though. Needs to be the same visual size as GitHub